fetch-csv/src/main.js

```diff
@@ -203,21 +203,32 @@ function getData(request) {
     valueSeparator = textQualifier + valueSeparator + textQualifier;
   }
 
-  var rows = contentRows.map(function(contentRow) {
-    if (textQualifier !== 'undefined') {
-      contentRow = contentRow.substring(1, contentRow.length - 1);
-    }
-    var allValues = contentRow.split(valueSeparator);
-    if (buildedFields.length !== allValues.length) {
-      sendUserError(
-        'Error during parsing content: the number of columns on each row is not respect'
-      );
-    }
-    var requestedValues = allValues.filter(function(value, index) {
-      return requestedFieldsIndex.indexOf(index) >= 0;
+  var rows = contentRows
+    .filter(function(contentRow) {
+      // Remove rows that are empty.
+      return contentRow.trim() !== '';
+    })
+    .map(function(contentRow, idx) {
+      if (textQualifier !== 'undefined') {
+        contentRow = contentRow.substring(1, contentRow.length - 1);
+      }
+      var allValues = contentRow.split(valueSeparator);
```

Comment:
I don't know much about CSV parsing, but is this safe? Don't you need to account for character `\e`scapes?

Reply:
I'm not sure, but I imagine it works in most scenarios. If we get issues related to this, we can fix it later.